collections/_utils.ts

```diff
@@ -23,3 +23,10 @@ export function filterInPlace<T>(
 
   return array;
 }
+
+/**
+ * Produces a random number between the inclusive `lower` and `upper` bounds.
+ */
+export function randomNumber(lower: number, upper: number) {
```

Comment:
Because it uses `Math.floor`, I guess it's better to name this `randomIntegerNumber`

Reply:
Or `randomInteger`, the number part is kind of redundant there :-)